config/routes.rb

```diff
@@ -68,5 +68,9 @@
   # Legacy URLs
   get "/prisoner-details", to: redirect("/prisoner")
 
-  get "/", to: redirect(ENV["GOVUK_START_PAGE"] || "https://www.gov.uk/prison-visits")
+  if Rails.env.production?
+    get "/", to: redirect(ENV["GOVUK_START_PAGE"] || "https://www.gov.uk/prison-visits")
```

Comment:
I think we can probably remove the `ENV` here and maybe move the start page URL to the Rails config? @threedaymonk / @tatyree any preferences?